### PR TITLE
feat: DAH-2337: Set up Account Pages

### PR DESCRIPTION
**Pull request by chadbrokaw**
[DAH-2337](https://sfgovdt.jira.com/browse/DAH-2337)

[DAH-2337]: https://sfgovdt.jira.com/browse/DAH-2337?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/assets/javascripts/shared/SharedService.js.coffee [hunk 1]
```diff
@@ -66,6 +66,18 @@ SharedService = ($http, $state, $window, $document) ->
     'dahlia.disclaimer':
       buildUrl: (state, params) -> Service._addLanguageAndParamsToUrl(params.lang, "/disclaimer")
       shouldRailsRoute: (isFirstLoad) -> !isFirstLoad
+    'dahlia.sign-in':
```
Comment (jimlin-sfgov) [inline app/assets/javascripts/shared/SharedService.js.coffee:69]: question: This is not an "account information" page and doesn't use the same env var either. Should this change be in a separate PR / Jira ticket?

Comment (chadbrokaw) [reply app/assets/javascripts/shared/SharedService.js.coffee:69]: Thanks for catching! I removed that line

**Review by jimlin-sfgov — APPROVED** [review]
lgtm
